Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -234,7 +234,7 @@ To test if your own test accounts are prone to brute forcing, append the correct
 
 > Tip: Perform these kinds of tests only at the very end of your penetration test. You don't want to lock out your account on the first day of testing and potentially having to wait for it to be unlocked. For some projects unlocking accounts might be more difficult than you think.  
 
-#### Login Throttling
+#### Login Throttling (MSTG‑AUTH‑6)
```

Comment:
what about moving this to a general "Testing User Login" section? There is one "Testing User Logout" section already. Consider adding other Login-relevant things there.

Reply:
Considering the previous 2 comments we would have removed all sub-sections from "Testing Authentication". As I wrote before, I would consider extraction all introductory parts from "Testing Authentication" and "Verifying that Appropriate Authentication is in Place" and put them in the chapter introduction as "general things about testing authentication". Then decide if a dedicated section is required for the remaining contents. At this point take a look at the outline again and consider referencing the things we will see on all "Testing..." sections. Also consider if the order of the "Testing..." sections matches the order of the things mentioned in the introduction. I would re-order the sections to match that order.

Reply:
Login Throttling is now part of the static analysis part of the password test case

Reply:
Maybe ping me in Slack to discuss further on this :-)